Add missing entry for UI WG meeting notes

gchhablani opened the pull request:
This PR adds the missing entry for UI WG meeting notes added in PR #170.
@chicken-biryani

chicken-biryani commented:
I think you accidentally deleted the one for the 19th and doubled the one for the 26th.

gchhablani commented:
For some reason it shows like I deleted the one for 26th and added it again. But it seems fine here: https://github.com/moja-global/About_moja_global/blob/82adc659aa20a50503fdb0ec7fcfbf7c6d0b1531/working-groups/user-interface/MeetingNotes.md?

chicken-biryani commented:
Huh. That's weird!
